### DISCUSSION TITLE
policy "Enable read access for all users" for table "objects" already exists

### DISCUSSION THREAD

**Discussion by x751685875** [supabase/supabase#11425] [Questions] [answered]
``` sql
create policy "Enable read access for all users" on storage.objects for
select using (bucket_id = 'avatars');

create policy "Enable read access for all users" on storage.objects for
select using (bucket_id = 'logos');
```

What should I do?

<img width="1037" alt="image" src="https://user-images.githubusercontent.com/23094123/210382366-94032073-54d0-4e3b-a965-8abe268dd9a9.png">

**Comment by GaryAustin1** [comment, marked as answer]
You just need to change the name of the policies.  
"Read Access for all on avatars", "Read Access for all on logos" or something along those lines.
